src/Workspaces/CSharpTest/Formatting/FormattingTests.cs

```diff
@@ -9861,6 +9861,74 @@ void Goo(R r)
 }", changedOptionSet: changingOptions);
         }
 
+        [Fact, Trait(Traits.Feature, Traits.Features.Formatting)]
+        [WorkItem(52413, "https://github.com/dotnet/roslyn/issues/52413")]
+        public async Task NewLinesForBraces_PropertyPatternClauses_Default()
+        {
+            await AssertFormatAsync(
+                @"
+class A
+{
+    public string Name { get; }
+
+    public bool IsFoo(A a)
+    {
+        return a is
+        {
+            Name: ""foo"",
+        };
+    }
+}",
+                @"
+class A
+{
+    public string Name { get; }
+
+    public bool IsFoo(A a)
+    {
+        return a is {
+            Name: ""foo"",
+        };
+    }
+}");
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.Formatting)]
+        [WorkItem(52413, "https://github.com/dotnet/roslyn/issues/52413")]
+        public async Task NewLinesForBraces_PropertyPatternClauses_NonDefault()
```

Comment:
💡 Can you also add a test showing that the pattern stays on one line if it starts that way?

```
[Theory, CombinatorialData]
public async Task NewLinesForBraces_PropertyPattern_SingleLine(bool option)
{
  var changingOptions = new OptionsCollection(LanguageNames.CSharp)
  {
    { NewLinesForBracesInObjectCollectionArrayInitializers, option },
  };

  // test here...
}
```
}